### DISCUSSION TITLE
Issues with BlueRetro Web config by Android phone

### DISCUSSION THREAD

**Discussion by ChaoTungTang** [darthcloud/BlueRetro#480] [Q&A] [answered]
Issues with BlueRetro Web config by Android  phone

Hi, thanks for reading

I use ESP32 DEVKIT V1 on N64,
After firmware update, can be used normally, rumble pak, Controller Pak are all fine

I use Android phone chrome to connect BlueRetro Web config 
unable to enter the BlueRetro System manager or others
the web always appear(in the end)
Argh! TypeError: Cannot read properties of undefined(reading 'indexof')

Is it a mobile phone problem?
Has anyone else had a similar experience?
![Screenshot_20221202-211115 jpg](https://user-images.githubusercontent.com/120081799/206376642-de0fa692-ed63-4903-bca4-0bd0f1eeb09d.jpg)

**Comment by darthcloud** [comment, marked as answer]
Try with the beta web config:
https://darthcloud.github.io/BlueRetroWebCfg_Beta/

**Reply by ChaoTungTang** [reply]
Thank you very much
the beta web config, can connect
But I haven't actually tested,  it can change the mapping or not
![1](https://user-images.githubusercontent.com/120081799/206845550-8a2fd868-30f0-409c-92c5-3e67fbe1288c.jpg)
